lib/ImagePrototypeInit.js

```diff
@@ -47,6 +47,19 @@
         this.__locked = false;
     };
 
+    Image.prototype.getMetadata = function() {
+        if (!this.__metadata || this.__metadata === "") {
+            return null;
+        } else {
+            return this.__metadata;
+        }
+    };
+
+    Image.prototype.setMetadata = function(data) {
+        this.__metadata = data;
```

Comment:
Good point, I think it would make sense to only allow metadata to be set to a string, so I added that check. 

I've also added the option to pass in `null` to setMetadata, which will cause the image to be saved without any tEXt chunks when `writeFile` is called.